website/docusaurus.config.ts

```diff
@@ -139,24 +139,39 @@ const config: Config = {
         ...defaultPresets,
       },
     ],
-    // [
-    //   '@docusaurus/plugin-content-docs',
-    //   {
-    //     id: 'guidelines',
-    //     path: '../docs/guidelines',
-    //     routeBasePath: 'guidelines',
-    //     ...defaultPresets,
-    //   },
-    // ],
-    // [
-    //   '@docusaurus/plugin-content-docs',
-    //   {
-    //     id: 'checklist',
-    //     path: '../docs/checklist',
-    //     routeBasePath: '/checklist/',
-    //     ...defaultPresets,
-    //   },
-    // ],
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'guidelines',
+        path: 'guidelines',
+        routeBasePath: 'guidelines',
+        async sidebarItemsGenerator({
+          defaultSidebarItemsGenerator,
+          numberPrefixParser,
+          item,
+          version,
+          docs,
+          categoriesMetadata,
+          isCategoryIndex,
+        }) {
+          // Example: return an hardcoded list of static sidebar items
+          return [
+            { type: 'doc', id: 'bottomsheet' },
+            { type: 'doc', id: 'bottomsheet' },
+          ];
+        },
+        ...defaultPresets,
+      },
+    ],
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'checklist',
+        path: 'checklist',
+        routeBasePath: '/checklist/',
```

Comment:
should this not be under the path `docs/checklist`, just for consistency with our other docs?

Reply:
Looking at the docusaurus documentation: https://docusaurus.io/docs/docs-multi-instance
They suggest to have unversioned doc in different folder